core/arch/x86/opcode.h

```diff
@@ -1350,19 +1350,32 @@ enum {
     /* 1181 */ OP_vcvtusi2ss,  /**< IA-32/AMD64 AVX-512 vcvtusi2ss opcode. */
     /* 1182 */ OP_vmovdqa32,   /**< IA-32/AMD64 AVX-512 vmovdqa32 opcode. */
     /* 1183 */ OP_vmovdqa64,   /**< IA-32/AMD64 AVX-512 vmovdqa64 opcode. */
-    /* 1184 */ OP_vmovdqu8,    /**< IA-32/AMD64 AVX-512 vmovdqu8 opcode. */
-    /* 1185 */ OP_vmovdqu16,   /**< IA-32/AMD64 AVX-512 vmovdqu16 opcode. */
-    /* 1186 */ OP_vmovdqu32,   /**< IA-32/AMD64 AVX-512 vmovdqu32 opcode. */
-    /* 1187 */ OP_vmovdqu64,   /**< IA-32/AMD64 AVX-512 vmovdqu64 opcode. */
+    /* 1184 */ OP_vmovdqu16,   /**< IA-32/AMD64 AVX-512 vmovdqu16 opcode. */
+    /* 1185 */ OP_vmovdqu32,   /**< IA-32/AMD64 AVX-512 vmovdqu32 opcode. */
+    /* 1186 */ OP_vmovdqu64,   /**< IA-32/AMD64 AVX-512 vmovdqu64 opcode. */
+    /* 1187 */ OP_vmovdqu8,    /**< IA-32/AMD64 AVX-512 vmovdqu8 opcode. */
```

Comment:
This re-ordering seems inferior: can we sort with -n?

Reply:
Done.